include/deal.II/hp/collection.h

```diff
@@ -252,6 +252,12 @@ namespace hp
     unsigned int
     size() const;
 
+    /**
+     * This is equivalent to <code>size() == 0</code>
+     */
+    bool
+    empty() const;
```

Comment:
As a general rule, try not to start sentences with "This" without a following noun (in documentation, theses, papers, etc.). That's because when just saying "This", it's often not clear what "This" actually refers to. This is particularly true if there is no previous sentence to which you refer. Here, you could say "This function" to make that clear.

Reply:
Goood point, thank you :)